lib/tasks/demo-build.js

```diff
@@ -25,14 +25,14 @@ function buildDemoSass(buildConfig) {
 				e.stack = '';
 				throw e;
 			}
-			return files.getBowerJson(buildConfig.cwd);
+			return files.getPackageJson(buildConfig.cwd);
 		})
-		.then(bowerConfig => {
+		.then(pkgJson => {
 			let prefixSass = '';
 			// If module has o-assets as a dependency, set local demos to use local assets
 			// to use the latest ones in the repo
-			if (bowerConfig && bowerConfig.dependencies && bowerConfig.dependencies['o-assets']) {
-				const moduleName = bowerConfig.name;
+			if (pkgJson && pkgJson.dependencies && pkgJson.dependencies['o-assets']) {
```

Comment:
I think we should check `peerDependencies` here? Or remove it it since [o-assets isn't used and will no longer work](https://github.com/Financial-Times/origami/pull/102)?

Reply:
yes, let's remove this as we won't be bringing o-assets over to npm for the forseeable future.

Reply:
maybe we should throw an error if a component depends on o-assets, explaining that o-assets is a tool for a bygone era and to talk to #origami-support about it

Reply:
I don't think we need to add an error since components aren't using o-assets anyway, we should deprecate o-assets though!

Reply:
https://github.com/Financial-Times/o-assets/pull/85